extension_service/alloydb.tests.cloudbuild.yaml

```diff
@@ -0,0 +1,93 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+  - id: Install dependencies
+    name: python:3.11
+    dir: extension_service
+    entrypoint: pip
+    args: ["install", "-r", "requirements.txt", "-r", "requirements-test.txt", "--user"]
+
+  - id: Create database
+    name: postgres
+    entrypoint: /bin/bash
+    env:
+      - "PGUSER=${_DATABASE_USER}"
```

Comment:
nit: probably worthwhile to put this in a user too

Reply:
What do you mean?

Reply:
I meant to say "in a secret"